resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
This doesn't seem like public API territory. Is all this intended to be set as part of event payloads by the source code, rather than being added by the internals of o-tracking?